src/app/containers/OnDemandHeading/StyledRadioHeadingContainer.jsx

```diff
@@ -3,10 +3,20 @@ import {
   GEL_SPACING_DBL,
   GEL_SPACING_QUAD,
 } from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
 import OnDemandHeading from '.';
 
 const StyledRadioHeadingContainer = styled(OnDemandHeading)`
-  padding: ${GEL_SPACING_QUAD} 0 ${GEL_SPACING_DBL};
+  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
```

Comment:
This component is extending `Headline` from `psammead-headings`. The padding being set that we need to overwrite is here https://github.com/bbc/psammead/blob/204f8523baee48047c8641b86176fc87b92cdf8b/packages/components/psammead-headings/src/index.jsx#L20

One alternative to my approach would be to directly overwrite the styles being applied at the 
 `${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} ` level.

Reply:
Opted for this approach!